.changeset/thirty-readers-attack.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/create-app': patch
+---
+
+Alignments of template with Backstage example app in monorepo
```

Comment:
I think it's alright to not describe the reordering and readme changes here, but I'd say it's best if we mention the `@spotify/prettier-config` version bump